slurp/src/main/webapp/scripts/init-firebase.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Initialize Firebase and its related product we use (Firestore and Firebase
+// Auth) using the provided Firebase project configuation.
```

Comment:
Although unlikely in our current plans that we'll use any other Firebase products, maybe remove the "(Firestore and Firebase Auth)" to keep it general?

Reply:
Good point. Done.